Source/WebCore/svg/SVGTransformValue.h

```diff
@@ -246,8 +246,10 @@ class SVGTransformValue {
     template<typename Number, typename... Numbers>
     static void appendFixedPrecisionNumbers(StringBuilder& builder, Number number, Numbers... numbers)
     {
-        if (builder.length() && builder[builder.length() - 1] != '(')
+        if (builder.length() && builder[builder.length() - 1] != '(') {
+            builder.append(',');
```

Comment:
I think we can do single 'builder.append' with `builder.append(', ')`.

Reply:
it fails. I tried. It seems to require a unique character. But indeed there should be a better way.

Reply:
Fair. My next would to search 'build.append(' in searchfox.org (wubkat) to see if we have usage of it with two spaces. I might have seen it in CSS area. Hence, I suggested it.

Reply:
```
In file included from /Users/karlcow/code/OpenSource/WebKitBuild/Debug/DerivedSources/WebCore/unified-sources/UnifiedSource97.cpp:1:
In file included from /Users/karlcow/code/OpenSource/Source/WebCore/accessibility/AccessibilityRenderObject.cpp:70:
In file included from /Users/karlcow/code/OpenSource/Source/WebCore/rendering/svg/legacy/LegacyRenderSVGShape.h:30:
In file included from /Users/karlcow/code/OpenSource/Source/WebCore/rendering/svg/legacy/LegacyRenderSVGModelObject.h:34:
In file included from /Users/karlcow/code/OpenSource/Source/WebCore/svg/SVGElement.h:25:
In file included from /Users/karlcow/code/OpenSource/Source/WebCore/svg/properties/SVGAnimatedPropertyImpl.h:42:
In file included from /Users/karlcow/code/OpenSource/Source/WebCore/svg/SVGTransformList.h:25:
In file included from /Users/karlcow/code/OpenSource/Source/WebCore/svg/SVGTransform.h:30:
/Users/karlcow/code/OpenSource/Source/WebCore/svg/SVGTransformValue.h:250:28: error: multi-character character constant [-Werror,-Wmultichar]
            builder.append(', ');
1 error generated.
In file included from /Users/karlcow/code/OpenSource/WebKitBuild/Debug/DerivedSources/WebCore/unified-sources/UnifiedSource96.cpp:6:
In file included from /Users/karlcow/code/OpenSource/Source/WebCore/accessibility/AccessibilityNodeObject.cpp:78:
In file included from /Users/karlcow/code/OpenSource/Source/WebCore/svg/SVGElement.h:25:
In file included from /Users/karlcow/code/OpenSource/Source/WebCore/svg/properties/SVGAnimatedPropertyImpl.h:42:
In file included from /Users/karlcow/code/OpenSource/Source/WebCore/svg/SVGTransformList.h:25:
In file included from /Users/karlcow/code/OpenSource/Source/WebCore/svg/SVGTransform.h:30:
/Users/karlcow/code/OpenSource/Source/WebCore/svg/SVGTransformValue.h:250:28: error: multi-character character constant [-Werror,-Wmultichar]
            builder.append(', ');
** BUILD FAILED **
```

Reply:
Oh maybe `builder.append(", ");` instead of `builder.append(', ');`

Reply:
> Oh maybe `builder.append(", ");` instead of `builder.append(', ');`

I am already trying it on my local build.

<img width="895" alt="image" src="https://github.com/WebKit/WebKit/assets/52317531/aa5c6c8f-cff8-409e-a7a4-e612a4a8f179">

Reply:
It compiles.

Reply:
> It compiles.

@karlcow: `builder.append(", ");`